diff --git a/src/jslib/corelib.js b/src/jslib/corelib.js
index 4f6a769d..0ec72ada 100644
--- a/src/jslib/corelib.js
+++ b/src/jslib/corelib.js
@@ -157,7 +157,7 @@ mergeInto(LibraryManager.library, {
     const initialNumberOfFunctionTableSlots = 15000;
     Module.numberOfFunctionTableSlotsToGrowBy = 15000;
 
-    const indirectFunctionTable = wasmExports['__indirect_function_table'];
+    const indirectFunctionTable = Module['asm']['__indirect_function_table'];
     const tableLengthBefore = indirectFunctionTable.length;
     indirectFunctionTable.grow(initialNumberOfFunctionTableSlots);
 
@@ -184,8 +184,8 @@ mergeInto(LibraryManager.library, {
 
       //console.log("compileAndPatchModule! moduleLength=%d", moduleLength);
       
-      const indirectFunctionTable = wasmExports['__indirect_function_table'];
-      const memory = wasmExports['memory'];
+      const indirectFunctionTable = Module['asm']['__indirect_function_table'];
+      const memory = Module['asm']['memory'];
 
       const table = new WebAssembly.Table({
         element: 'anyfunc',
@@ -281,7 +281,7 @@ mergeInto(LibraryManager.library, {
   wasmReleaseBlock: function(block) {
     if (Module.blockToCompiledFunctionIndexes[block]) {
 
-      const indirectFunctionTable = wasmExports['__indirect_function_table'];
+      const indirectFunctionTable = Module['asm']['__indirect_function_table'];
       const compiledFunctionIndexes = Module.blockToCompiledFunctionIndexes[block];
       
       while (compiledFunctionIndexes.length > 0) {
